packages/models/tests/integration/models-test.js

```diff
@@ -85,6 +84,8 @@ module('Integration | Models', function(hooks) {
         type: 'pages'
       },{
         type: 'authors'
+      },{
+        type: 'fields'
```

Comment:
Why is this necessary?

Reply:
When I was debugging I kept running into errors about duplicated fields but might have been related to the fact that I was running the tests on server mode. I'll check if it's really needed, probably not.

Reply:
so as @jorgelainfiesta mentioned, if you're running this in server mode (i.e. restarting test every time you save the file) then the test will fail on the second execution of the test with an error along the lines of `Could not create field with id "blah" because it already exists`

This indicates that the types described in the destroy definition were not expansive enough 👍 so this should get committed

Reply:
ok, cool 👍